examples/chart/teleport-cluster/values.yaml

```
@@ -700,6 +700,10 @@ service:
 # See https://goteleport.com/docs/architecture/tls-routing/#working-with-layer-7-load-balancers-or-reverse-proxies-preview for details.
 ingress:
   enabled: false
+  # useExisting indicates to the chart that you are providing your own ingress.
```

```suggestion
  # useExisting indicates to the chart that you are managing your own ingress.
```

providing gives the impression that you need to provide the ingress to the chart

---

Hi @tigrato , same comment is probably valid also at docs/pages/reference/helm-reference/teleport-cluster.mdx at line 1902 in this branch.